server/resp/src/main/java/org/infinispan/server/resp/commands/string/APPEND.java

```diff
@@ -0,0 +1,30 @@
+package org.infinispan.server.resp.commands.string;
+
+import io.netty.channel.ChannelHandlerContext;
+import org.infinispan.server.resp.commands.Resp3Command;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespRequestHandler;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * @link https://redis.io/commands/append/
+ * @since 15.0
+ */
+public class APPEND extends RespCommand implements Resp3Command {
+   public APPEND() {
+      super(3, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
+                                                      ChannelHandlerContext ctx,
+                                                      List<byte[]> arguments) {
+      return handler
```

Comment:
check arguments size 

@jabolina I think we need a general validaiton of number of arguments, what do you think ?